spec/01-unit/19-hybrid/03-compat_spec.lua

```diff
@@ -417,8 +417,39 @@ describe("kong.clustering.compat", function()
             name = "correlation-id",
             instance_name = "my-correlation-id"
           },
+          plugin3 = {
+            id = "00000000-0000-0000-0000-000000000003",
```

Comment:
The reason you have to do this here is that in the `off` strategy, the order in which plugins are returned is determined by their `id`. Blueprints generate a new `uuid` if this field is not provided.